pkg/kubelet/kuberuntime/helpers.go

```diff
@@ -210,7 +210,18 @@ func toKubeRuntimeStatus(status *runtimeapi.RuntimeStatus) *kubecontainer.Runtim
 			Message: c.Message,
 		})
 	}
-	return &kubecontainer.RuntimeStatus{Conditions: conditions}
+	retHandlers := make(map[string]kubecontainer.RuntimeHandler)
+	for _, h := range handlers {
+		supportsUserns := false
+		if h.Features != nil {
+			supportsUserns = h.Features.SupportsUserNamespaces
```

Comment:
Should be set to false unless the feature gate is enabled

Reply:
we agreed on this behaviour during the KEP review